### PR TITLE
No beamform checks

### PR DESCRIPTION
**Pull request by NickSwainston**
Removed all the beamforming from check_known_pulsars.py and replaced it with a simple file check. Also removed the searching and upgraded the Nextflow pulsar search modules so they can perform targeted searchs.

Made all the Nextflow scripts executable so they can be added to the path. Added help scripts so how to use them is clearer. Made a default work dir to be /astro/mwaops/${USER}/${params.obsid}_work.

### CODE REVIEW

File: scripts/check_known_pulsars.py [hunk 1]
```diff
@@ -408,6 +418,10 @@ def beamform_and_fold(obsid, DI_dir, cal_obs, args, psrbeg, psrend,
     vcstools_version: string
         OPTIONAL - The version of vcstools to use. Default = 'master'
     """
+    base_dir = "{0}/{1}/dpp_pointings/".format(product_dir, obsid)
```
Comment (keegansmith21) [inline scripts/check_known_pulsars.py]: should use os.path.join when making paths to avoid extra forward slashes

File: scripts/pulsars_in_fov.py [hunk 2]
```diff
@@ -171,21 +173,27 @@ def find_pulsars_in_fov(obsid, psrbeg, psrend):
...
-    pulsar_pointing_list = pulsar_pointing_list + temp[1]
+    fermi_list = get_sources_in_fov(obsid, 'Fermi', fwhm)
+    pulsar_search_name_list = pulsar_search_name_list + fermi_list[0]
+    pulsar_search_pointing_list = pulsar_search_pointing_list + fermi_list[1]
 
     # Find all of the points of interest candidates
     #-----------------------------------------------------------------------------------------------
-    temp = get_sources_in_fov(obsid, 'POI', fwhm)
-    pulsar_name_list = pulsar_name_list + temp[0]
-    pulsar_pointing_list = pulsar_pointing_list + temp[1]
```
Comment (keegansmith21) [inline scripts/pulsars_in_fov.py:182]: not using temp <3

Comment (NickSwainston) [reply scripts/pulsars_in_fov.py:182]: You highlighted something I removed preemptively to keep you happy!

File: scripts/check_known_pulsars.py (hunk 1 above)
Comment (NickSwainston) [reply scripts/check_known_pulsars.py]: Done

**Pull request merged**